src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -417,6 +417,19 @@ namespace AppInstaller::Manifest
         return "Unknown"sv;
     }
 
+    std::string_view UnsupportedArgumentToString(UnsupportedArgumentEnum unsupportedArg)
+    {
+        switch (unsupportedArg)
+        {
+        case UnsupportedArgumentEnum::Log:
+            return "log"sv;

```

Comment:
Why not include the `--` so that you don't have to prepend it to print the message?

Reply:
I decided not to include the `--` in case the actual name of the unsupported argument needed to be used elsewhere.